DUnitX.Assert.pas

```diff
@@ -48,21 +49,22 @@ interface
   Assert = class
   private
     class var fOnAssert: TProc;
-    class var fTestFailure: ExceptClass;
```

Comment:
D'oh - did you understand what these fields were for? This was to not couple the assert class with the framework but now you went back how it was before by hardcoding the exceptions.